### DISCUSSION TITLE
License is for Edge but extension build is for Chrome

### DISCUSSION THREAD

**Discussion by LOSjr03** [aclap-dev/video-downloadhelper#878] [Help and Q&A]
### Your question

 I just bought the license but it's not letting me download, so I went into the settings and it said, "License is for Edge but extension build is for Chrome". How can I fix this problem?

### Addon & CoApp version and details

 I just bought the license but it's not letting me download, so I went into the settings and it said, "License is for Edge but extension build is for Chrome". How can I fix this problem?

**Comment by jcfrog** [comment]
Hi,
You can't use Edge premium on Chrome.
If you want to change your premium: https://github.com/aclap-dev/video-downloadhelper/wiki/I-purchased-the-wrong-premium

**Reply by samirpa02** [reply]
I have same problem but im using Edge. Any help please?

**Reply by jcfrog** [reply]
Maybe you installed Chrome extension on Edge.
I recommend to uninstall the extension and reinstall it from there: https://microsoftedge.microsoft.com/addons/detail/video-downloadhelper/jmkaglaafmhbcpleggkmaliipiilhldn

**Reply by samirpa02** [reply]
Exactly! Thank you so much!!
